tests/h/search/index_test.py

```diff
@@ -48,72 +46,6 @@ def test_it_indexes_presented_annotation(
         )
         assert indexed_ann == presenter.asdict.return_value
 
-    def test_it_can_index_an_annotation_with_no_document(
-        self, factories, index_annotations, get_indexed_ann
-    ):
-        annotation = factories.Annotation.build(document=None)
-
-        index_annotations(annotation)
-
-        assert get_indexed_ann(annotation.id)["document"] == {}
-
-    def test_it_indexes_the_annotations_document_web_uri(
-        self, factories, index_annotations, get_indexed_ann
-    ):
-        annotation = factories.Annotation.build(
-            document=factories.Document.build(
-                web_uri="https://example.com/example_article"
-            )
-        )
-
-        index_annotations(annotation)
-
-        # *Searching* for an annotation by ``annotation.document`` (e.g. by
-        # document ``title`` or ``web_uri``) isn't enabled.  But you can
-        # retrieve an annotation by ID, or by searching on other field(s), and
-        # then access its ``document``. Bouncer
-        # (https://github.com/hypothesis/bouncer) accesses h's Elasticsearch
-        # index directly and uses this ``document`` field.
-        assert (
-            get_indexed_ann(annotation.id)["document"]["web_uri"]
-            == "https://example.com/example_article"
-        )
-
-    def test_it_can_index_an_annotation_with_a_document_with_no_web_uri(
-        self, factories, index_annotations, get_indexed_ann
-    ):
-        annotation = factories.Annotation.build(
-            document=factories.Document.build(web_uri=None)
-        )
-
-        index_annotations(annotation)
-
-        assert "web_uri" not in get_indexed_ann(annotation.id)["document"]
```

Comment:
This is again kind of a test for `DocumentSearchIndexPresenter` which handles omitting `web_uri` from the Elasticsearch document if the document has no `web_uri`.

Again, I'm not how it could _not_ be possible for it to be able to index this? Unless `web_uri` were made required in the Elasticsearch schema